docs/2.0/resources.md

```diff
@@ -567,3 +567,81 @@ end
 
 This tells Avo which resources you use and stops the eager-loading process on boot-time.
 This means that other resources that are not declared in this array will not show up in your app.
+
+:::option self.pagination
+<VersionReq version="2.45" />
+This feature is designed for managing pagination. For example on large tables of data sometimes count is inefficient and unnecessary.
+
+By setting `self.pagination[:type]` to `:countless`, you can disable the pagination count on the index page.
+
+This is especially beneficial for large datasets, where displaying the total number of items and pages may have some performance impact.
+
+```ruby
+# As block:
+self.pagination = -> do
+  {
+    type: :default,
+    size: [1, 2, 2, 1],
+  }
+end
+
+# Or as hash:
+self.pagination = {
+  type: :default,
+  size: [1, 2, 2, 1],
+}
+```
+
+The exposed pagination setting above have the default value for each key.
+
+### `type`<br><br>
```

Comment:
Can these be displayed as `:::option`? Maybe they will stand out better?

Reply:
![CleanShot 2023-11-17 at 17 03 55@2x](https://github.com/avo-hq/avodocs/assets/1334409/63aa8cf7-131f-42f4-9412-7de981ae23e4)

Reply:
@adrianthedev i tried it but that will make the option visible on the right lateral sidebar and it looks like "type" and "size" becomes an resource option. Would be nice to have "nested options" so we can declare :::option inside an :::option

Reply:
ok. leave it like that.
Shouldn't we add the docs to 3.0 too?